bika/lims/browser/aggregatedanalyses/aggregatedanalyses.py

```diff
@@ -58,15 +58,36 @@ def __init__(self, context, request, **kwargs):
             }
         self.review_states = [
             {'id': 'default',
-             'title':  _('Results pending'),
+             'title': _('Assignment pending'),
+             'transitions': [{'id': 'submit'},
+                             {'id': 'cancel'},
```

Comment:
Remove `cancel` transition. At this state (Results pending + unassigned), only `submit` transition should be allowed